ecleankernel/file.py

```diff
@@ -96,6 +97,25 @@ def __init__(self,
         super().__init__(path, KernelFileType.KERNEL)
         self.internal_version = self.read_internal_version()
 
+    def decompress_lzma(self, data):
+        results = []
+        while True:
+            decomp = LZMADecompressor(FORMAT_AUTO, None, None)
```

Comment:
These args look like the defaults. Am I missing something? If not, then I guess:

```suggestion
            decomp = LZMADecompressor()
```